src/main/java/co/inventrosoft/springboottask/configuration/AppConfiguration.java

```diff
@@ -0,0 +1,14 @@
+package co.inventrosoft.springboottask.configuration;
+
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class AppConfiguration {
+    @Bean
+    public ObjectMapper mapper() {
```

Comment:
This is the best way to build _ObjectMapper_.
org.springframework.http.converter.json.Jackson2ObjectMapperBuilder